Pcap++/header/PcapLiveDevice.h

```diff
@@ -425,59 +447,63 @@ namespace pcpp
 		 * @return True if packet was sent successfully. False will be returned in the following cases (relevant log error is printed in any case):
 		 * - Device is not opened
 		 * - Packet length is 0
-		 * - Packet length is larger than device MTU
 		 * - Packet could not be sent due to some error in libpcap/WinPcap/Npcap
+		 * This method will not check the Mtu of the packet
 		 */
 		bool sendPacket(const uint8_t* packetData, int packetDataLength);
 
 		/**
 		 * Send a parsed Packet to the network
 		 * @param[in] packet A pointer to the packet to send. This method treats the packet as read-only, it doesn't change anything in it
+		 * @param[in] checkMtu Whether the length of the packet's payload should be checked against the MTU. If enabled this comes with a small performance penalty.
```

Comment:
ditto - I'd also mention that the default value is `false`

Reply:
will do